app/src/components/app.rs

```diff
@@ -29,6 +30,25 @@ use yew_router::{
     BrowserRouter, Switch,
 };
 
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(js_namespace = darkmode)]
+    fn toggleDarkMode(doSave: bool);
```

Comment:
> Save is done through what? Cookies, local storage?

[ref for the js function](https://github.com/vinorodrigues/bootstrap-dark-5/blob/main/docs/darkmode.js.md#indarkmode)
It looks like it could use either cookies or local storage, depending on user consent. Either way, it is stored under the key `bs.prefers-color-scheme` (customizable)